plugins/http_extract/src/http_extract/component.py

```
@@ -0,0 +1,580 @@
+            await self._apply_rate_limit()
+
+        return dataframes
+
+    async def _make_request(
+        self, url: str, params: dict[str, Any] | None = None
+    ) -> Any:
+        """Выполнение HTTP запроса."""
+        response, _ = await self._make_request_with_headers(url, params)
+        return response
+
+    async def _make_request_with_headers(
```

Ruff (0.12.2)

`_make_request_with_headers` is too complex (11 > 10)

(C901)